src/lib/core/components/visualizations/implementations/ScatterplotVisualization.tsx

```diff
@@ -460,19 +462,14 @@ function ScatterplotViz(props: VisualizationProps) {
           response.completeCasesTable
         );
 
-      // For the abundance app, the overlay vocabulary is within the comptuedVariableMetadata.
       const overlayVocabulary =
-        computation?.descriptor.type === 'abundance' &&
-        entities &&
-        response.scatterplot.config.computedVariableMetadata?.collectionVariable
-          ?.collectionVariableDetails
-          ? response.scatterplot.config.computedVariableMetadata?.collectionVariable?.collectionVariableDetails.map(
-              (variableDetails) => variableDetails.variableId
-            )
-          : fixLabelsForNumberVariables(
-              overlayVariable?.vocabulary,
-              overlayVariable
-            );
+        response.scatterplot.config.computedVariableMetadata?.collectionVariable?.collectionVariableDetails?.map(
```

Comment:
i wonder if we should have a check here to see if the overlay var is a provided var? Otherwise anytime we return a collection var as part of `computedVariableMetadata` we'll modify the overlayVocabulary. 

To be fair, i can't think of an instance where we'd return a collection var as part of the response to scatter and not use it on the overlay, but adding the check that the overlay var is provided may make the logic more human-digestible?